CHANGELOG.md

```diff
@@ -23,6 +23,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Added rollback feature to `Databricks` source.
 - Changed all Prefect logging instances in the `sources` directory to native Python logging.
 - Changed `rm()`, `from_df()`, `to_df()` methods in `S3` Source
+- Changed `to_df()` method in `sap_rfc` Source
```

Comment:
You actually performed two actions:
- Added new method `_exclude_whitespaces()` to the `sap_rfc` source 
- Modified `to_df()` to trim the data using `_exclude_whitespaces()`

Could you please reflect it in the `CHANGELOG` and PR description as well?

Reply:
[📝 Added Method in SAP Source](https://github.com/dyvenia/viadot/pull/627/commits/760c8e11c291158afff9e6a22fa20ca65dda52c0)

Reply:
Could you please add some details in the second line to prevent getting same entries couple of times? 

For example, we might need to change one more thing in `to_df()` and would get another generic entry 'Changed xxx method in yyy source' in the CHANGELOG. After some time it's hard to find out which feature was added in particular entry.

Reply:
Could you please add some details in the second line to prevent getting same entries couple of times? 

For example, we might need to change one more thing in `to_df()` and would get another generic entry 'Changed xxx method in yyy source' in the CHANGELOG. After some time it's hard to find out which feature was added in particular entry.

Reply:
[📝 Added further documentation in CHANGELOG](https://github.com/dyvenia/viadot/pull/627/commits/4efe6e542503c4904259ecfc82cb70f086732c90)